src/OrchardCore/OrchardCore.AsModule/OrchardCore.AsModule.props

```diff
@@ -8,6 +8,7 @@
   <PropertyGroup>
     <ModuleManifest>Name: $(MSBuildProjectName)</ModuleManifest>
     <ExcludedFiles>**\*.cs;*.csproj*;obj\**;bin\**;Assets.json;Assets\**;Properties\**;*.props;*.targets;.vs\**</ExcludedFiles>
+    <Temp Condition="'$(Temp)' == ''">..\..\..\temp</Temp>
   </PropertyGroup>
```

Comment:
Yes, good idea and just checked that this property is also passed to the referenced projects when they are built.

So, modules use the Temp folder to store their assets, then notice that the Temp folder is also used by the main app through `AsBundle.targets`, this to retrieve all stored assets. So, defining the Temp variable in AsModule / AsTheme would not be sufficient.

But, as you suggested, using a global property through the command line to define the Temp property should work.

Also, would be good to get rid of the Temp folder as suggested in #707, for this i will continue the discusion on your issue #708 where i will have some questions.